test/core/elements.wast

```diff
@@ -0,0 +1,163 @@
+;; Test the element section
```

Comment:
Nit: most tests follow the convention to be named after the keyword of the construct they test. Therefore, can we name the file `elem`?